android/faq_android_studio.md

```diff
@@ -0,0 +1,73 @@
+# Android Studio FAQ
+
+## Build Issues
+
+_What do I do if building in Android Studio fails (and the same version is building for everyone else)?_
+
+_Try the following in this order:_
+
+1. Verify the desired Build Variant is selected
+2. Clean project then rebuild
+3. Sync project with Gradle files
+4. Invalidate Caches and Restart
```

Comment:
After we get rid of SNAPSHOTs, I don't think items 4, 6, 7, 8, or 10 are necessary - I've never had to do any of these things.

Reply:
yea, if that's gonna happen soon, we can definitely simplify this